setup.cfg

```diff
@@ -1,98 +1,5 @@
-[metadata]
-name = jwst
-description = Library for calibration of science observations from the James Webb Space Telescope
-long_description = Library for calibration of science observations from the James Webb Space Telescope
-long_description_content_type = text/plain
-author = JWST calibration pipeline developers
-license = BSD-3-Clause
-url = https://github.com/spacetelescope/jwst
-project_urls =
-    Tracker = https://github.com/spacetelescope/jwst/issues
-    Documentation = https://jwst-pipeline.readthedocs.io/en/stable/
-    Source Code = https://github.com/spacetelescope/jwst
-classifiers =
-    Intended Audience :: Science/Research
-    Topic :: Scientific/Engineering :: Astronomy
-    License :: OSI Approved :: BSD License
-    Operating System :: MacOS :: MacOS X
-    Operating System :: POSIX
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3 :: Only
-        Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-
-[options]
-zip_safe = False
-python_requires = >=3.8
-setup_requires =
-    setuptools_scm
-install_requires =
-    asdf>=2.12.0
-    astropy>=5.0.4
-    BayesicFitting>=3.0.1
-    certifi==2022.5.18.1
-    crds>=11.16.5
-    drizzle>=1.13.4
-    gwcs>=0.18.0
-    jsonschema>=4.0.1
-    numpy>=1.20
-    photutils>=1.4.0
-    psutil>=5.7.2
-    poppy>=1.0.2
-    pyparsing>=2.2.1
-    requests>=2.22
-    scikit-image>=0.17.2
-    scipy>=1.6.0
-    spherical-geometry>=1.2.22
-    stcal>=1.0
-    stdatamodels>=0.4.3,<1.0
-    stpipe>=0.4.1,<1.0
-    stsci.image>=2.3.5
-    stsci.imagestats>=1.6.3
-    tweakwcs>=0.7.2
-    asdf-astropy>=0.2.1
-    wiimatch>=0.2.1
-
-[options.extras_require]
-aws =
-    stsci-aws-utils>=0.1.2
-docs =
-    packaging
-    matplotlib
-    sphinx
-    sphinx-asdf>=0.1.1
-    sphinx-astropy
-    sphinx-automodapi
-    sphinx-rtd-theme
-    stsci-rtd-theme
-    mistune~=0.8.4
-sdp =
-    jplephem>=2.9
-    pymssql>=2.1.6
-    pysiaf>=0.13.0
-test =
-    ci-watson>=0.5.0
-    codecov>=1.6.0
-    colorama>=0.4.1
-    flake8>=3.6.0
-    getch>=1.0.0
-    pytest>=6.0.0
-    pytest-cov>=2.9.0
-    pytest-doctestplus>=0.10.0
-    pytest-openfiles>=0.5.0
-    requests_mock>=1.0
-
-[options.entry_points]
-asdf.extensions =
-    jwst_pipeline = jwst.transforms.integration:get_extensions
-asdf_extensions =
-    jwst_datamodel = jwst.datamodels.extension:DataModelExtension
-asdf.resource_mappings =
-    jwst_pipeline = jwst.transforms.integration:get_resource_mappings
-stpipe.steps =
-    jwst = jwst.stpipe.integration:get_steps
-pytest11 =
-    report_crds_context = pytest_crds.plugin
+# this file exists to support an editable PEP517 install and for `flake8` (https://github.com/PyCQA/flake8/issues/234)
```

Comment:
You might be able to move the `flake8` configuration to tox.ini or .flake8, but I don't think those are any better than setup.cfg.  I'm not convinced the `sphinx` configuration in here is doing anything anyway.  That code has been there forever, many doc build systems and CI systems ago.

Reply:
Good idea, I moved the configuration to `.flake8`